dex.html

```diff
@@ -451,7 +455,45 @@ <h4>RAPIDOC Newsletter</h4><br>
       </svg>
     </div>
   </div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.12.2/gsap.min.js"></script>
+  <script>
+    // First, make sure to include the GSAP library in your HTML file:
+
+// Then add this JavaScript code:
+
+document.addEventListener("DOMContentLoaded", function() {
+  const images = document.getElementsByTagName('img');
+
+  Array.from(images).forEach(img => {
+    img.addEventListener("mousemove", function(event) {
+      const rect = img.getBoundingClientRect();
```

Comment:
Can you move this code to utils folder? If not create a new folder and add it. Reusable code makes codebase cleaner

Reply:
sure